scripts/validate_string_concatenation.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+"""
+Check where there is a string that needs to be concatenated.
+"""
+
+import os
+import sys
+import token
+import tokenize
+
+FILE_EXTENTIONS_TO_CHECK = [".py", ".pyx"]
+
+
+def main():
+    path = sys.argv[1]
+
+    if not os.path.exists(path):
+        raise ValueError("Please enter a valid path, to a file/directory.")
+
+    if os.path.isfile(path):
+        # Means that the given path is of a single file.
+        sys.exit(is_concatenated(path))
+
+    status_codes = set()
+    # Means that the given path is of a directory.
+    for subdir, _, files in os.walk(path):
+        for file_name in files:
+            ext = os.path.splitext(os.path.join(subdir, file_name))[1]
+            if ext in FILE_EXTENTIONS_TO_CHECK:
+                status_codes.add(is_concatenated(os.path.join(subdir, file_name)))
+
+    if 1 in status_codes:
+        sys.exit(1)
+
+    sys.exit(0)
+
+
+def is_concatenated(file_path):
+    """
+    Checking if the file containing strings that needs to be concatenated.
+
+    Parameters
+    ----------
+    file_path : str
+        File path pointing to a single file.
+
+    Returns
+    -------
+    int
```

Comment:
Feels like a bool would be a more appropriate return type if this function is only ever going to return 0/1

Reply:
I did also thought so, but I am relying on an exit status, so int feels more native.